dragonfly/world/block.go

```diff
@@ -189,6 +189,7 @@ func RegisterBlock(b Block, s BlockState) error {
 	if !ok {
 		return fmt.Errorf("block state returned is not currently registered (%+v)", s)
 	}
+	fmt.Println(s.Name, rid)
```

Comment:
Make sure to remove this.

Reply:
Removed, completely forgot about this lmao